package.json

```diff
@@ -110,13 +110,14 @@
     "@escape.tech/mookme": "2.4.1",
     "@redux-devtools/remote": "0.8.0",
     "@types/express": "^4.17.21",
-    "@web3modal/ethers": "4.1.0",
+    "@wagmi/core": "2.16.7",
     "dotenv": "^16.4.5",
     "express": "^4.19.2",
     "pg": "8.8.0",
     "pino": "^9.1.0",
     "react-native-google-cast": "4.6.2",
-    "elliptic": "6.6.1"
+    "elliptic": "6.6.1",
+    "valtio": "1.13.2"
```

Comment:
needed to hoist this, was causing problems otherwise

Reply:
It was _so fun_ to try to debug why I got some weird esoteric error about it.. turns out the proxy state was being written in one version, and attempted to be consumed in the other, so it was missing! Really fun times